odoo/sql_db.py

```diff
@@ -100,6 +101,64 @@ def check(f, self, *args, **kwargs):
         raise psycopg2.OperationalError('Unable to use a closed cursor.')
     return f(self, *args, **kwargs)
 
+class Savepoint:
+    """ Reifies an active breakpoint, allows :meth:`BaseModel.savepoint` users
```

Comment:
I guess you mean `BaseCursor` instead of `BaseModel`.

Reply:
Yes look like it.